packages/direflow-component/src/config/config-overrides.ts

```diff
@@ -10,24 +11,35 @@ export = function override(config: TConfig, env: string, options?: IOptions) {
   const chunkFilename = options?.chunkFilename || 'vendor.js';
 
   const overridenConfig = {
-    ...addWelcomeMessage(config, env),
+    ...addEntries(config, env),
     module: overrideModule(config.module),
     output: overrideOutput(config.output, { filename, chunkFilename }),
     optimization: overrideOptimization(config.optimization, env),
     resolve: overrideResolve(config.resolve),
     plugins: overridePlugins(config.plugins, env, { filename, chunkFilename }),
+    externals: overrideExternals(config.externals, env),
   };
 
   return overridenConfig;
 };
 
-function addWelcomeMessage(config: TConfig, env: string) {
+function addEntries(config: TConfig, env: string) {
+  let entry: string[] = [];
+
+  if (env === 'development') {
+    entry = [...config.entry, resolve(__dirname, './dist/config/welcome.js')];
+  }
+
   if (env === 'production') {
-    return config;
+    const [pathIndex] = config.entry;
+    const entryLoaderFile = fs.readFileSync(resolve(__dirname, './dist/config/entryLoader.js'), 'utf8');
+    const entryLoaderTemplate = handlebars.compile(entryLoaderFile);
+    const entryLoaderFileNew = entryLoaderTemplate({ pathIndex });
+    fs.writeFileSync(resolve(__dirname, './dist/config/entryLoader.js'), entryLoaderFileNew);
+
+    entry = [resolve(__dirname, './dist/config/entryLoader.js')];
```

Comment:
im having concerns about this. seems like we could maybe come up with a cleaner solution :)

Reply:
Remember that this part of the code is executed in a Node process as a part of the Webpack build.
So this is not going to be run in the browser.
Can you elaborate on which parts that concerns you?